NOTICE.txt

```diff
@@ -10,30 +10,37 @@ This product includes software developed by Aurelius (http://thinkaurelius.com/)
  * Marko A. Rodriguez
  * Stephen Mallette
  * Pavel Yaskevich
+ * Others as listed in CONTRIBUTORS.txt
```

Comment:
Just pushed the update with your original requests resolved before seeing this. I'll put in an issue for this. For the record I do kind of like specifically acknowledging the original Aurelius team members separate from other contributors.